index.php

```diff
@@ -533,6 +535,18 @@ public function createBackup() {
 		$this->silentLog('[info] end of createBackup()');
 	}
 
+	/**
+	* Returns a link to the changelog
+	* @return string
+	*
+	*/
+	private function getChangelogURL() {
+		$this->silentLog('[info] getChangelogURL()');
+		$changelogWebsite = "https://nextcloud.com/changelog/";
+		$changelogURL = $changelogWebsite . '#' . str_replace('.','-', substr($this->getConfigOption('version'),0,-2));
```

Comment:
I would add a single whitespace after the comma. Just my personal preference but easier to read.

`str_replace('.', '-', substr($this->getConfigOption('version'), 0, -2)` 

Most examples from https://docs.nextcloud.com/server/14/developer_manual/general/codingguidelines.html looks like that.

Reply:
changed -- thank you!